bob-backend/src/pearl/disk_controller.rs

```diff
@@ -400,10 +400,10 @@ impl DiskController {
     }
 
     pub(crate) async fn put(
-        &self,
+        self: Arc<Self>,
```

Comment:
Why this is needed? `&self` would be better

Reply:
This function is called inside spawn(), so it is not possible to pass simple reference to another thread

Reply:
Ok, I will use async block to do so